core/src/tx.rs

```diff
@@ -509,7 +514,7 @@ mod tests {
         let mut chain = err.chain();
         assert_eq!(
             chain.next().unwrap().to_string(),
-            "Failed to accept transaction"
+            "Limits verification failed"
```

Comment:
This is a `ui` test. I feel we shouldn't mix them in with the regular tests. Perhaps this could be improved by using an `enum` instead of `&str`

Reply:
yeah, the issue is that we rely on `eyre` error reports too often and then you have no choice but compare strings. Like in this case for `AcceptedTransaction::from_transaction`. Ideally, we should return error reports only when reporting errors outside rust (logging, cli, network). I will not change this now, this should be done systematically across codebase